contracts/gas-snapshots/keystone.gas-snapshot

```diff
@@ -78,12 +78,12 @@ CapabilityRegistry_UpdateNodesTest:test_RevertWhen_P2PIDEmpty() (gas: 27313)
 CapabilityRegistry_UpdateNodesTest:test_RevertWhen_SignerAddressEmpty() (gas: 26996)
 CapabilityRegistry_UpdateNodesTest:test_RevertWhen_UpdatingNodeWithoutCapabilities() (gas: 25560)
 CapabilityRegistry_UpdateNodesTest:test_UpdatesNodeInfo() (gas: 158847)
-KeystoneForwarder_ReportTest:test_Report_ConfigVersion() (gas: 1805070)
-KeystoneForwarder_ReportTest:test_Report_SuccessfulDelivery() (gas: 157545)
-KeystoneForwarder_ReportTest:test_RevertWhen_AlreadyProcessed() (gas: 155666)
-KeystoneForwarder_ReportTest:test_RevertWhen_AnySignatureIsInvalid() (gas: 86609)
-KeystoneForwarder_ReportTest:test_RevertWhen_AnySignerIsInvalid() (gas: 118685)
-KeystoneForwarder_ReportTest:test_RevertWhen_ReportHasDuplicateSignatures() (gas: 94760)
+KeystoneForwarder_ReportTest:test_Report_ConfigVersion() (gas: 1805086)
```

Comment:
Looks like the gas here actually increased? I used a variable since caching signatures.length seems to save on gas